History.md

```diff
@@ -199,6 +199,10 @@
   and `html` files have been updated to use this new API. Read more on the
   [Wiki page](https://github.com/meteor/meteor/wiki/Build-Plugins-API).
 
+* There is a new API for adding assets to packages, `api.addAssets`. It works
```

Comment:
OK, I think it's helpful to link to the upgrade guide from the top of the 1.2 History section anyway though.